.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,91 @@
+name: 🐛 Bug Report
+description: Report a bug encountered while using Cluster API
+labels: kind/bug
+body:
+  - type: textarea
+    id: problem
+    attributes:
+      label: What steps did you take and what happened?
+      description: |
+        Please provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner.
+        If this matter is security related, please follow the guidelines described in https://github.com/kubernetes-sigs/cluster-api/blob/main/SECURITY_CONTACTS
+      placeholder: "[A clear and concise description on how to REPRODUCE the bug.]"
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected
+    attributes:
+      label: What did you expect to happen?
+    validations:
+      required: true
+
+  - type: textarea
+    id: repro
+    attributes:
+      label: How can we reproduce it (as minimally and precisely as possible)?
+    validations:
+      required: false
+
+  - type: textarea
+    id: additional
+    attributes:
+      label: Anything else you would like to add?
+      placeholder: "[Miscellaneous information that will assist in solving the issue.]"
+
+  - type: textarea
+    id: capiVersion
+    attributes:
+      label: Cluster-api version
+      value: |
+        <details>
+
+        </details>
+    validations:
+      required: true
+
+  - type: textarea
+    id: kubeVersion
+    attributes:
+      label: Kubernetes version
+      value: |
+        <details>
+
+        ```console
+        $ kubectl version
+        # paste output here
+        ```
+
+        </details>
```

Comment:
Q: is there a way to provide instructions in a way that the user is not forced to clean them up when adding a value? what about using placeholder?

Reply:
Sure, I will try to use a placeholder if possible or simply remove the instructions since it is obvious that output should be under the command :)

Reply:
I moved the "$ kubectl version" to the placeholder